web/blueprints/user/__init__.py

```diff
@@ -562,8 +562,9 @@ def get_membership_or_404(id: int) -> Membership:
         abort(404)
     return membership
 
-@bp.route('/<int:user_id>/end_membership/<int:membership_id>')
-@access.require('groups_change_membership')
+
+@bp.route("/<int:user_id>/end_membership/<int:membership_id>", methods=["GET", "POST"])
+@access.require("groups_change_membership")
 def end_membership(user_id: int, membership_id: int) -> ResponseReturnValue:
```

Comment:
Please add frontend tests for this.
This is not hard, just look at the existing ones.
Hit me up if you have any questions!